index.js

```diff
@@ -251,6 +251,24 @@ class Client {
         }, callback);
     }
 
+    /**
+      * All updates within the given interval
```

Comment:
really sorry, but only the first line of the comment block was supposed to be indented by 1 space. (so the first stars line up) eg:

```
/**
 * blah
 * blah
 */